test/integration/application.spec.ts

```diff
@@ -28,7 +28,7 @@ describe('Application', () => {
   it('should allow GET /', async () => {
     const result = await request(server).get('/');
     expect(result.status).to.equal(HttpStatus.OK);
-    expect(result.text).to.equal('home');
```

Comment:
Don't we want to confirm that some text from the expected page is present?

Reply:
This is asking for the home page right? I recall seeing something returned int he document.